useful-scripts/misc-test-utils/src/utils/guid.ts

```diff
@@ -0,0 +1,84 @@
+/* eslint-disable import/prefer-default-export */
+export class Guid {
```

Comment:
Wouldn't ignore this lint - we can make this a default export. We'd also want to the name the file `Guid.ts` as class files should typically be PascalCased.